schemas/context/message.schema.json

```diff
@@ -38,38 +38,7 @@
                 "$ref": "action.schema.json#"
```

Comment:
@Yannick-Malins can you change the `additionalProperties.anyOf` to `oneOf` - the semantics are virtually the same, but the code generation via QuickType differs (it'll merge properties with `anyOf` as you could be multiple of the referenced types at once, where we intend it to be `oneOf` the referenced types)

Reply:
@Yannick-Malins The change I'm after is on line 36 above. Otherwise, this is good to merge, apart from a Changelog entry and the `$id` I flagged above.